aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -253,6 +256,34 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 				Computed:    true,
 				Description: "Private IP address of HA transit gateway.",
 			},
+			"bgp_polling_time": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      50,
+				Description:  "BGP route polling time. Unit is in seconds. Valid values are between 10 and 50.",
+				ValidateFunc: validation.IntBetween(10, 50),
+			},
+			"prepend_as_path": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of AS numbers to populate BGP AP_PATH field when it advertises to VGW or peer devices.",
+				Elem: &schema.Schema{
+					Type:         schema.TypeInt,
+					ValidateFunc: validateIntASN(),
+				},
+			},
+			"local_as_number": {
+				Type:         schema.TypeInt,
```

Comment:
Change type to String instead of Int. String is used for ASN in all other resources.